### DISCUSSION TITLE
肝完主線感想

### DISCUSSION THREAD

**Discussion by Andrea0126** [share121/inter-knot#5412] [绝区零]
劇情我覺得還挺不錯的，但沒想到策劃是真的不留一點走格子在主線裡面，全部都是打小怪，老實說完全沒有繩匠的沉浸感，而且打小怪的時候還要忙著看劇情對話，有時候我根本沒注意到，只能說拜託之後主線至少加一點走格子吧😭
![Screenshot_2024-09-25-23-45-06-303_com HoYoverse Nap](https://github.com/user-attachments/assets/f7c3d247-4955-4805-9c09-cb7b3b0f88ba)
題外話這張圖真好看

**Comment by cpu644** [comment]
确实。格子至少对话更好搞，并且现在全是打怪确实有点怪了

**Comment by LootDust** [comment]
+1，全战斗也没有绳匠感觉了。说真的把走格子画面暗、卡动画和部分关卡弱引导的问题修一修会好很多，也不至于那么多人骂走格子了

**Reply by HKLHaoBin** [reply]
有一句话说得好，他反馈的并不是他自己真正想要的。

**Comment by TimhGyeus** [comment]
吐槽下莱特没实装但有莱特打斗的剧情...gacha也就这样了，啥都要让步卖卡

**Reply by HKLHaoBin** [reply]
也不能这么说，零号空洞剧情不也一样？柳还没有实装，但也有打斗画面，且柳的卡池在1.3版本，根本就没有刻意为了卖卡一说，反倒是你，引战？
是何居心？

**Reply by TimhGyeus** [reply]
我有啥居心，主线做到那里沉浸感被打破了，出戏了，吐槽下。明明是莱特作为常胜冠军和别的常胜冠军对决，试用给我塞两个别人，这要是莱特实装了给试用多好。这就引战了？绝区零不是gacha？我就觉得卡池安排影响了剧情战斗演出，没这一说我也说了。
另我觉得这期没走格子还算说得过去，视角大多在代理人身上，这露天车载hdd没内味了
